InterGalacticCurrencyConvertor/src/main/java/numerals/InterGalacticNumerals.java

```diff
@@ -0,0 +1,43 @@
+package main.java.numerals;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+/**
+ * Keeps track of InterGalactic Numerals and its conversion to Roman Numerals.
+ */
+public class InterGalacticNumerals {
+	static HashMap<String, String> mMap = new HashMap<>();
+	private static InterGalacticNumerals mInstance = new InterGalacticNumerals();
+	private InterGalacticNumerals() {}
+	public static InterGalacticNumerals getInstance() {
+		return mInstance;
+	}
+	public void addRNConversion(String inIGN, String inRNum) {
```

Comment:
nit: Rename parameters to:
ignSymbol, romanSymbol

Reply:
updated